icaruscode/CRT/CRTUtils/CRTT0MatchAlg.h

```diff
@@ -59,9 +59,31 @@
 #include "TVector3.h"
 #include "TGeoManager.h"
 
+//New
+#include "larreco/RecoAlg/Cluster3DAlgs/Cluster3D.h"
+#include "Eigen/Core"
+#include "Eigen/Dense"
+#include "Eigen/Eigenvalues"
+#include "Eigen/Geometry"
+#include "Eigen/Jacobi"
+
+
 
 namespace icarus{
 
+  struct tpc_cand {
+
+	bool found_match = false;
+	recob::Track thistrack;
+	TVector3 PCA_start_pos{DBL_MAX,DBL_MAX,DBL_MAX};
```

Comment:
In general, I'd advise against initializing values using DBL_MAX, DBL_MIN, etc. These values are OS-dependent, so someone on another machine might try "if PCA_start_pos[0] == DBL_MAX" and get different behavior. Maybe it's fine in this case since we assume this is running on gpvms, but I'd still suggest picking values that are not OS-dependent. Note that if you initialize the directions to {0}, you'll need to check the denominator in the DCA calculation to make sure it's not zero!